yb-voyager/cmd/importData.go

```diff
@@ -595,11 +594,10 @@ func splitFilesForTable(filePath string, t string, taskQueue chan *SplitFileImpo
 			numLinesInThisSplit += 1
 		}
 
-		if linesWrittenToBuffer {
-			line = fmt.Sprintf("\n%s", line)
+		if line != "" {
+			line = fmt.Sprintf("%s\n", line)
```

Comment:
This will cause a new memory allocation. Why can't we just write "\n" on its own:
```
length, err := bufferedWriter.WriteString(line)
if err != nil {
  ...
}
_, err = bufferedWriter.WriteString("\n")
if err != nil {
  ...
}
length++ // Account for \n.
```

Reply:
This should be good way. Let me do the changes and test it.

Reply:
done